app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -146,7 +146,10 @@ class HomeFragment : Fragment() {
     private val store: BrowserStore
```

Comment:
nit: can we shorten the commit message summary? See my comment in the other review.